<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeworkPage</class>
 <widget class="QWidget" name="HomeworkPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="titleEdit">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="contentEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>DDL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="ddlEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>还剩</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="dayShow">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>天</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="hourShow">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>时</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="minuteShow">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>分</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="saveBtn">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="confirmFinish">
        <property name="text">
         <string>确认完成</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="confirmUnfinish">
        <property name="text">
         <string>确认未完成</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
